open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/janus-connector/pom.xml

```diff
@@ -97,6 +97,26 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <!--<plugin>-->
```

Comment:
what's the intent here? Is it just work in progress or never used?

Reply:
It is required to build the janus connector jar, but I commented it out to test whether it was the reason behind the failing build.

Reply:
fair enough - so it will go back in?

Reply:
Done